pyproject.toml

```diff
@@ -142,7 +142,7 @@ tests/codegen/snapshots/
 '''
 
 [tool.pytest.ini_options]
-addopts = "-s --emoji --mypy-ini-file=mypy.ini --benchmark-disable"
+addopts = "--emoji --mypy-ini-file=mypy.ini --benchmark-disable"
```

Comment:
any reason why this was removed? 😊

Reply:
I'm an old school "print debugger" guy =P, and `-s` avoids tests being able to capture stdout/stderr to show those, and instead they get displayed while running the tests.

I'm not sure if there was an specific reason for having `-s` in here, but I thought it would be ok to remove. I'm fine with keeping it if you prefer :)

Reply:
no that's fine, I actually thought it was doing the opposite 🤣